amti/utils/workers.py

```diff
@@ -0,0 +1,40 @@
+""" Module for worker management functions """
+import boto3
+import click
+import csv
+from typing import List, Optional
+
+def create_batches(items: List, n=100) -> List:
+    """ Create generator that splits items into batches of size n. """
+    for i in range(0, len(items), n):
+        yield items[i:i + n]
+
+def read_workerids_from_file(file: click.Path) -> List:
```

Comment:
It looks like the expected file format for this function is one worker id per line, possibly with a header. If that's the case, it might be simpler to just require these files not have headers and read the file directly:

```python
with open(worker_ids_path, 'r') as worker_ids_file:
    worker_ids = [worker_id.strip() for ln in worker_ids_file]
```

If it is important to read CSVs, I'd suggest having `read_workerids_from_file` take the file-like object as an argument. The advantage over passing around the path is that then the CLI commands can open the files using `click.open_file` which supports things like using `-` to represent standard out. This feature enables `amti` to be used in unix pipelines.